Functions.Templates/Templates/IoTHubServiceBusQueueTrigger-CSharp/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "defaultFunctionName": "IoTHubServiceBusQueueTriggerCSharp",
+  "description": "$ServiceBusQueueTrigger_description",
+  "name": "IoT Hub (Service Bus Queue)",
+  "language": "C#",
+  "category": [
+    "$temp_category_IoTHub"    
+  ],
+  "categoryStyle": "iot",
+  "enabledInTryMode": false,
+  "userPrompt": [
+    "connection",
+    "accessRights",
+    "queueName"
+  ],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.ServiceBus",
+      "version": "3.0.1"
```

Comment:
Use version 3.0.0. Currently portal prompts for an update for any difference in versions. We don't have update experience in portal yet.